common/views/components/WorkTags/WorkTags.js

```diff
@@ -0,0 +1,91 @@
+import styled from 'styled-components';
+import { spacing, font, classNames } from '../../../utils/classnames';
+import NextLink from 'next/link';
+import type { NextLinkType } from '@weco/common/model/next-link-type';
+
+export type WorkTagType = {|
+  query: string,
+  textParts: string[],
+  linkAttributes: NextLinkType,
+|};
+
+const WorkTag = styled.div`
+  border-radius: 3px;
+  text-decoration: none;
+  padding: 0.2em 0.5em;
+  transition: color 250ms ease, background 250ms ease;
+`;
+
+type Props = {
+  tags: WorkTagType[],
+};
+
+const WorkTags = ({ tags }: Props) => {
```

Comment:
Good to eep it as "WorkTag" - but there's probably a more descriptive name, maybe `SegmentedTag`... but we can wait to see what comes out of the wash.

Reply:
Thinking about it though, we don't have tags, so why not call these tags?

Reply:
Fibs, we do, but they actually act the same so might be worth thinking about merging once we've done the testing here.